src/modules/commander/Commander.cpp

```diff
@@ -1876,6 +1877,23 @@ Commander::run()
 				}
 			}
 
+			if (param_man_arm_gesture != PARAM_INVALID && param_rc_map_arm_sw != PARAM_INVALID) {
```

Comment:
Would be better to do this before the `check for unsafe Airmode settings` check above, so we don't change airmode unnecessarily

Reply:
I didn't think about that dependency 🤦‍♂️ 
It's now before: https://github.com/PX4/PX4-Autopilot/commit/45ec969957d7590454def7aa42aedc5785f6383b#diff-e4b611e75eba3844661619867ee06befbcf230aced354a14f167f7d0ebbca8e0R1919